src/sentry/search/django/backend.py

```diff
@@ -185,6 +185,10 @@ def _build_queryset(self, project, query=None, status=None, tags=None,
                     params['datetime__lt'] = date_to
 
             event_queryset = Event.objects.filter(**params)
+
+            if query:
+                queryset = queryset.filter(message__icontains=query)
```

Comment:
Don't you want this on `event_queryset`? The `Group` already should have this applied.